SpacemanDMM.toml

```diff
@@ -34,3 +34,4 @@ hide_invisible = [
 
 [map_renderer.render_passes]
 smart-cables = false
+gravity-gen = false
```

Comment:
Might be worth leaving a comment in the file as to why this is the case

Reply:
Done!